db/structure.sql

```diff
@@ -672,7 +711,8 @@ CREATE TABLE ctgov.users (
     confirmed_at timestamp without time zone,
     confirmation_sent_at timestamp without time zone,
     db_activity integer,
-    last_db_activity timestamp without time zone
+    last_db_activity timestamp without time zone,
+    admin boolean DEFAULT false
```

Comment:
I think you accidentally included the changes from the branch where you added admin to users. Please do a `bin/rake db:structure:dump` to refresh the structure file